### PR TITLE
New package: ShiningLight.OpenSSL version 3.2.1

### LINKED ISSUE #140716: [Package Request]: ShiningLight.OpenSSL
### How can we help?

None

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://slproweb.com/products/Win32OpenSSL.html
Publisher: Shining Light Productions
Package Name: OpenSSL
Description: Installs Win64 OpenSSL v3.2.1 (Recommended for software developers by the creators of [OpenSSL](https://www.openssl.org/)).
Package Version: v3.2.1
Installer URL: https://slproweb.com/download/Win64OpenSSL-3_2_1.msi

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [X] Resolve: #140716

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/140760)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

Nearly every EXE is asking for this dependency.

File: manifests/s/ShiningLight/OpenSSL/3.2.1/ShiningLight.OpenSSL.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ShiningLight.OpenSSL
+PackageVersion: 3.2.1
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{1E36B3DA-676B-4994-92BE-382B0647F657}'
+Installers:
```
Comment (stephengillie) [inline manifests/s/ShiningLight/OpenSSL/3.2.1/ShiningLight.OpenSSL.installer.yaml:9]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
[Policy] Needs-Author-Feedback

File: manifests/s/ShiningLight/OpenSSL/3.2.1/ShiningLight.OpenSSL.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: ShiningLight.OpenSSL
+PackageVersion: 3.2.1
+PackageLocale: en-US
+Publisher: Shining Light Productions
+PackageName: OpenSSL 3.2.1 (64-bit)
```
Comment (stephengillie) [inline manifests/s/ShiningLight/OpenSSL/3.2.1/ShiningLight.OpenSSL.locale.en-US.yaml:8]:
```suggestion
PackageName: OpenSSL 3.2.1 (64-bit)
PackageUrl: https://slproweb.com/products/Win32OpenSSL.html
```

**Review by stephengillie — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**